src/components/view/MealEditorView/MealEditorView.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
```

Comment:
Can I see a demo of how this works when it is dismissed? Trying to imagine how it would be configured in vb.
Appearance wise, It looks identical to MealEditor

Reply:
Yes, for sure, I can show you a demo.  The modal dialog just closes, though.

This view component is lightweight wrapper around the editor component, so it doesn't add any functionality other than defining the actions to take when things like save and cancel occur.  In view builder, these handlers are configured as action definitions on the editor component node, so the view component is not necessary.

I'm planning to remove this view and the `GlucoseChartView` now that view builder support has been implemented.  I just left them here for the time being so that folks could see how the new components are meant to fit together.